src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file(s) command";
+
+        public const string AppDescription = "Command to add, remove and list projects from the solution (SLN) file.";
+
+        public const string AppHelpText = "Projects to add to the solution.";
+
+        public const string CmdSlnFile = "<SLN_FILE>";
```

Comment:
I guess it's safe to leave it in then even if we don't use it. Since today is the loc deadline. But if you leave that, could you add <PROJECT> to CommonLocalizableStrings? Since I know I'm not localizing it currently...